src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -80,7 +80,19 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
     const stringIndex = funcTable.name[funcIndex];
     const functionCall = stringTable.getString(stringIndex);
     const name = isJS ? functionCall : getFunctionName(functionCall);
-    copy(name);
+    return name;
+  }
+
+  lookupFunctionOnSearchfox(): void {
+    const name = this._getFunctionName();
+    window.open(
+      `http://searchfox.org/mozilla-central/search?q=${name}`,
```

Comment:
Some changes needed here:
1. use `https`
2. use [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) for the parameter `name`.

Reply:
Ah, yes, of course :) Thanks